tracing-subscriber/tests/subscriber_filters/option.rs

```diff
@@ -0,0 +1,40 @@
+use super::*;
+use tracing_subscriber::{filter, prelude::*, Subscribe};
+
+fn filter<S>() -> filter::DynFilterFn<S> {
```

Comment:
small nit, since it took me a second to understand what was happening here:
```suggestion
fn filter_out_everything<S>() -> filter::DynFilterFn<S> {
```

Reply:
Ditto copied from `boxed.rs`.